stix_shifter_modules/qradar/stix_translation/json/aql_event_fields.json

```diff
@@ -36,8 +36,12 @@
     "credibility",
     "relevance",
     "geographiclocation as geographic",
+    "\"CRE Name\" as creName",
+    "\"CRE Description\" as creDescription",
     "creEventList",
+    "rulename(creEventList) as rulenames",
     "domainid",
-    "LOGSOURCETYPENAME(devicetype) as devicetype"
+    "LOGSOURCETYPENAME(devicetype) as devicetypename",
+    "devicetype"
```

Comment:
Probably doesn't matter but the capitalization is not consistent on the keys:
cre**N**ame/cre**D**escriptions vs rule**n**ames etc.

Reply:
Yeah, it doesn't matter for the AQL query but I may downcase them to make it consistent.